src/lib/libcmd/meson.build

```diff
@@ -1,8 +1,12 @@
-libcmd_files = ['context.c', 'cmdinit.c', 'stty.c']
+libcmd_files = ['context.c', 'cmdinit.c' ]
+
+libcmd_files += [ 'basename.c', 'stty.c', 'head.c' ]
```

Comment:
It's not obvious why these modules aren't just part of the previous list. Which means it probably needs a comment.